api/ProfileName/[profileAddress].ts

```diff
@@ -0,0 +1,59 @@
+import assert from 'assert';
+
+import type { VercelRequest, VercelResponse } from '@vercel/node';
+import { isAddress } from 'ethers/lib/utils';
+
+import { adminClient } from '../../api-lib/gql/adminClient';
+import { errorResponse, NotFoundError } from '../../api-lib/HttpError';
+
+const CACHE_SECONDS = 60 * 5;
+export default async function handler(req: VercelRequest, res: VercelResponse) {
+  try {
+    let address: string | undefined;
+    if (typeof req.query.profileAddress == 'string') {
+      address = req.query.profileAddress;
```

Comment:
is there a use case for the address to be wrapped in an array? Or should we simplify and expect the profileAddress to always just be in the first `if` case?

Reply:
I agree, I will fix that